python/py-readme_renderer/Portfile

```diff
@@ -2,32 +2,43 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
-PortGroup           github 1.0
-
-github.setup        pypa readme_renderer 26.0
 
 name                py-readme_renderer
-platforms           darwin
+version             30.0
+revision            0
 supported_archs     noarch
 license             apache
 maintainers         {aronnax @lpsinger} openmaintainer
 
 description         Safely render long_description/README files in Warehouse
-long_description    ${description}
+long_description    {*}${description}
+
+homepage            https://github.com/pypa/readme_renderer
 
-checksums           rmd160  67451b077be105f5f1beb1ca997249d54dd6f86f \
-                    sha256  9852c2c926d82444a99b818042e9d296616db3bb9084a14aef1940161dd502d8 \
-                    size    23800
+checksums           rmd160  017c4292b1f7b7cc51b9e5ddc89769cd547f55bb \
+                    sha256  8299700d7a910c304072a7601eafada6712a5b011a20139417e1b1e9f04645d8 \
+                    size    27065
 
-python.versions     27 36 37 38
+python.versions     27 37 38 39
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
+    if {${python.version} eq {27}} {
```

Comment:
no need for `{` `}` around the `27`

Reply:
Renee, would it be preferable to use a numeric comparison via `==`, as done in other Python ports? Or is a string comparison the optimal approach?

Reply:
both options work and both options are used in ports... I *think* that a numeric comparison with `==` as you suggest is the "correct" way but at least in the past I/we haven't been strict on requesting a change either way. If we want to do so moving forward it would be good to document it somewhere (or perhaps it already is and I just don't know about it).

Reply:
I recall being corrected about using `tcl` string comparisons once in Portfiles, and it made sense to me because `tcl` is string-based and string comparison is more efficient than converting the strings to integers, then comparing the integers.

Reply:
`eq` is 20% faster than `==`: https://wiki.tcl-lang.org/page/eq